components/service/firefox-accounts/build.gradle

```diff
@@ -33,9 +33,11 @@ dependencies {
 
     // Parts of this dependency are typealiase'd or are otherwise part of this module's public API.
     api Dependencies.mozilla_fxa
+    implementation Dependencies.mozilla_sync_manager
 
     // Observable is part of public API of the FxaAccountManager.
     api project(':support-base')
+    implementation project(':service-glean')
```

Comment:
Why do we wan to include glean here? If it is for telemetry reasons I think that what we need to do is to emit facts as [this pr is doing](https://github.com/mozilla-mobile/android-components/pull/4494)

Reply:
Interesting. I wasn't aware of `facts`. Yes, this is for telemetry, but I've since moved into into a new `support` module.

Let's come back to this sometime after it lands and re-consider the current approach.